src/mesh/index_derivs.cxx

```diff
@@ -423,8 +423,13 @@ class FFTDerivativeType {
     ASSERT2(bout::utils::is_Field3D<T>::value); // Should never need to call this with Field2D
 
     // Only allow a whitelist of regions for now
-    ASSERT2(region == "RGN_ALL" || region == "RGN_NOBNDRY"
-            || region == "RGN_NOX" || region == "RGN_NOY");
+    ASSERT2(region == "RGN_ALL" || region == "RGN_NOBNDRY" || region == "RGN_NOX"
+            || region == "RGN_NOY" || region == "RGN_INNER_X" || region == "RGN_OUTER_X"
+            || region == "RGN_LOWER_INNER_Y" || region == "RGN_UPPER_INNER_Y"
+            || region == "RGN_LOWER_OUTER_Y" || region == "RGN_UPPER_OUTER_Y"
+            || region == "RGN_LOWER_Y" || region == "RGN_UPPER_Y" || region == "RGN_BNDRY"
+            || region == "RGN_WITH_XBNDRIES" || region == "RGN_WITH_YBNDRIES"
+            || region == "RGN_WITH_BNDRIES");
```

Comment:
Is the whitelist here actually needed? Would `theMesh->getRegion2D(region)` not throw an exception anyway if the region was not allowed?

Reply:
Good point, I think that should be sufficient